asciimatics/screen.py

```diff
@@ -1347,6 +1348,49 @@ def wrapper(cls, func, height=None, catch_interrupt=False, arguments=None,
         finally:
             screen.close(restore)
 
+    class Session():
```

Comment:
Declaring a class inside a class is a little bit off the beaten path and so may well confuse people.  All in all, I think I'd rather than it wasn't nested and simply called `ScreenSession`.

Reply:
I wasn't sure about how to do it, but I do think that would be a good change